notebooks/Amsterdam.ipynb

```diff
@@ -0,0 +1,1848 @@
+       "      <th>259427</th>\n",
+       "      <td>259428</td>\n",
+       "      <td>Acer campestre</td>\n",
+       "      <td>10.5</td>\n",
+       "      <td>3</td>\n",
+       "      <td>4.968974</td>\n",
+       "      <td>52.298392</td>\n",
+       "      <td>7.62</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>259428</th>\n",
+       "      <td>259429</td>\n",
+       "      <td>Acer pseudoplatanus</td>\n",
+       "      <td>21.0</td>\n",
+       "      <td>7</td>\n",
+       "      <td>4.969281</td>\n",
+       "      <td>52.298189</td>\n",
+       "      <td>17.78</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>259429</th>\n",
+       "      <td>259430</td>\n",
+       "      <td>Carpinus betulus</td>\n",
+       "      <td>10.5</td>\n",
+       "      <td>3</td>\n",
+       "      <td>4.967459</td>\n",
+       "      <td>52.300042</td>\n",
+       "      <td>7.62</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>259430</th>\n",
+       "      <td>259431</td>\n",
+       "      <td>Acer campestre</td>\n",
+       "      <td>13.5</td>\n",
+       "      <td>4</td>\n",
+       "      <td>4.967483</td>\n",
+       "      <td>52.300068</td>\n",
+       "      <td>10.16</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "            id              species  height  radius       lng        lat  \\\n",
+       "259426  259427  Acer pseudoplatanus    21.0       7  4.968736  52.298498   \n",
+       "259427  259428       Acer campestre    10.5       3  4.968974  52.298392   \n",
+       "259428  259429  Acer pseudoplatanus    21.0       7  4.969281  52.298189   \n",
+       "259429  259430     Carpinus betulus    10.5       3  4.967459  52.300042   \n",
+       "259430  259431       Acer campestre    13.5       4  4.967483  52.300068   \n",
+       "\n",
+       "          dbh  \n",
+       "259426  17.78  \n",
+       "259427   7.62  \n",
+       "259428  17.78  \n",
+       "259429   7.62  \n",
+       "259430  10.16  "
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def get_average_height(height_string):\n",
+    "    \"\"\"\n",
+    "    Extracts the average height from a string of the format 'X tot Y m'.\n",
+    "    \"\"\"\n",
+    "    matches = re.findall(r'\\d+', height_string)\n",
+    "    return sum(map(int, matches)) / len(matches)\n",
+    "\n",
+    "df_pop['height'] = df_pop['height'].apply(get_average_height)\n",
+    "\n",
+    "df_pop.tail()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 1.4 Map LAT,LNG to x,y POS on MESA Grid."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df_pop = df_pop.apply(latlng_to_xy, axis=1)"
```

Comment:
@oguzhan I think this method might be wrong. The maps you generated of A'dam seem to me to be on their side...
The Latitude is the north/south measure, and Longitude, the east/west measure. So I think line 55 here should be inverted, right?
<img width="646" alt="image" src="https://github.com/lucidmindsai/gus/assets/110916952/68490526-db1d-463e-a892-0f0170ea3c0d">

Reply:
you think the method from utm returns the results inverted or we assigned it to the wrong vars after the calculation?

Reply:
I think longitude should be X, and latitude should be Y

Reply:
Oh, never mind, the UTM method returns "easting" and "northing" in that order, which would be correct. I was just grasping for reasons why the plots looked so weird, but I guess we would need more trees to recognise the city